common/app/common/models/LocationSelector.java

```diff
@@ -48,10 +49,11 @@ public void setCountry(final List<SelectableData> country) {
                     }
                     return selector;
                 }).collect(toList());
+        return (countrySelector.size() > 1) ? countrySelector : emptyList();
```

Comment:
For some weird lit implementation size is expensive, so I would prefer `!countrySelector.isEmpty()`, also it does not require a magic number.